src/SignalR/clients/ts/FunctionalTests/ts/WebWorkerTests.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+import { ENDPOINT_BASE_URL } from "./Common";
+
+// On slower CI machines, these tests sometimes take longer than 5s
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 10 * 1000;
+
+describe("WebWorkers", () => {
+    it("can use SignalR client", (done) => {
+        if ((window as any).Worker) {
```

Comment:
`if (typeof window !== "undefined" && (window as any).Worker) {`
Also, you can put the if around the whole describe block so it just doesn't run the test